develop-docs/sdk/expected-features/index.mdx

```diff
@@ -225,7 +225,7 @@ Whenever possible, avoid adding the attachment altogether if taking the screensh
 
 ## Before-Send Hook
 
-Hook called with the event (and on some platforms the hint) that allows the user to decide whether an event should be sent or not.  This can also be used to further modify the event. This only works for `error` events. For `transactions` it is recommended to have `beforeSendTransaction` implemented in SDKs.
+Hook called with the event (and on some platforms the hint) that allows the user to decide whether an event should be sent or not.  This can also be used to further modify the event. This only works for `error` events. For `transactions` it is recommended to have `beforeSendTransaction` implemented in SDKs. To understand when you should call this in your SDK, please refer to the [filter ordering of session](/sdk/telemetry/sessions/#filter-order).
```

Comment:
The docs talk only about error filtering. Let's point it out

Reply:
I'm sorry, I don't get your comment. What change are you proposing, @stefanosiano?

Reply:
Just explicitly mention the errors in the filter ordering, so that we make it more clear to the user that transactions don't follow the same order
```suggestion
Hook called with the event (and on some platforms the hint) that allows the user to decide whether an event should be sent or not.  This can also be used to further modify the event. This only works for `error` events. For `transactions` it is recommended to have `beforeSendTransaction` implemented in SDKs. To understand when you should call this in your SDK, please refer to the [errors filter ordering of session](/sdk/telemetry/sessions/#filter-order).
```